client/src/helpers/bus.js

```diff
@@ -0,0 +1,3 @@
+import Vue from "vue"
+
+export default new Vue()
```

Comment:
Why do we need this? Shouldn't it just be part of the `app.vue` file?

Reply:
This simplifies showing the snackbar. Anywhere you include this file you can just call 
bus.$emit('updateSnack', "Your notification")
to update the snackbar with the new text and it will show.
without this you have to to this.$root.$emit(...) and it would only work in .vue files